lib/mirage.ml

```diff
@@ -466,7 +466,8 @@ let fat_block ?(dir=".") ?(regexp="*") () =
   impl @@ object
     inherit block_conf block_file as super
 
-    method configure i =
+    method packages = Key.pure [ package "fat-filesystem" ]
```

Comment:
when it is only required during build time, you can use the `~build:true` argument to `package` (see example usage in crunch). if also needed at runtime, do as above.